libr/include/r_util/r_str.h

```diff
@@ -21,6 +21,12 @@ typedef enum {
 
 typedef int (*RStrRangeCallback) (void *, int);
 
+typedef struct { ut8 ch; ut8 hx; } charset;
+
+typedef struct r_charset_t {
+	charset custom_charset[256];
```

Comment:
why 256? magic numbers are bad.

Reply:
@crowell There are potentially 255 different chars in an opcodes.